_data/boards.yml

```diff
@@ -1,3 +1,26 @@
+-
+  title: LS2088A
+  label: Arm 64bit
+  format: left
+  image: /assets/images/LS2088A-RDB-BD1.jpg
+  text: >
+      The QorIQ® LS2088A reference design board provides a comprehensive
+      platform that enables design and evaluation of the product (QorIQ LS2088A
+      processor). This RDB comes pre-loaded with a board support package (BSP)
```

Comment:
We could do away with the "This RDB comes pre-loaded..... to the end of the sentence.